lib/dish.rb

```diff
@@ -0,0 +1,8 @@
+class Dish
```

Comment:
This class might be unnecessary, as it doesn't do anything that couldn't be done with a hash: `{ name: "Pizza", price: 7.00 }`